src/Game.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Hyperdrive;
+
+use Hyperdrive\Panels\FinalScorePanel;
+use Hyperdrive\Panels\MainPanel;
+use Hyperdrive\Player\CreatePlayer;
+use Hyperdrive\Player\Player;
+use Symfony\Component\Config\Definition\Exception\Exception;
+
+class Game
+{
+    protected ?Player $player;
+
+    public function start(): void
+    {
+        $this->player = CreatePlayer::create();
```

Comment:
`createPlayer` to doskonała nazwa dla metody ale dla klasy nie za bardzo. 

Może `PlayerCreator`, `PlayerGenerator` lub `PlayerFactory` to lepsza nazwa dla tej klasy?

Czy metoda `create` musi być statyczna w tym przypadku?

Reply:
Racja, takie nazwy dla klasy bardziej jej odpowiadają, a w takim przypadku metoda `create` niekoniecznie musi być statyczną. Taka klasa `PlayerFactory` mogłaby posłużyć do stworzenia więcej niż tylko jednego gracza.